packages/expo-cli/src/commands/publish/publishAsync.ts

```diff
@@ -150,6 +154,25 @@ function assertValidReleaseChannel(releaseChannel?: string): void {
   }
 }
 
+function assertExpoUpdatesInstalled(projectDir: string, sdkVersion?: string): void {
+  // before sdk 44, expo-update was included in with the expo module
+  if (sdkVersion && semver.lt(sdkVersion, '44.0.0')) {
```

Comment:
if someone is using expo-updates prior to sdk 44 in a bare project they would have react-native-unimodules instead of the expo package, and in that case their sdkVersion would be undefined. I'm not sure why someone would run `expo publish` in that context, though.

i'd prefer to skip any validation we're doing here in the context that the sdkVersion is undefined for now because we're specifically trying to address sdk 44+ defaulting to no expo-updates in this pr.